hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/utils/EvmTokenUtil.java

```diff
@@ -162,7 +162,8 @@ public static void extractFees(
 
             evmCustomFee.setFixedFee(fixedFee);
             evmCustomFees.add(evmCustomFee);
-        } else if (customFee.getFractionalFee().getMinimumAmount() > 0) {
+        } else if (customFee.getFractionalFee().getMinimumAmount() > 0
```

Comment:
nit: in a future PR we should make this check within the custom fee object e.g. `customFee.hasAmount()`

Reply:
Makes sense, agreed.